pages.pl/common/7z.md

```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> Archiwizator plików o wysokim stopniu kompresji.
+> Więcej informacji: <https://www.7-zip.org/>.
+
+- Zarchiwizuj plik lub katalog:
+
+`7z a {{archived.7z}} {{path/to/file_or_directory}}`
```

Comment:
Here and pretty much in all other files, you should translate the text inside `{{ }}`, too, unless it's a list of concrete options like `{{zip|gzip|bzip2|tar}}`.

Example: `archived` from `{{archived.7z}}` should be translated, `{{path/to/file_or_directory}}` should be translated completely.

Reply:
Now I see. Thanks for your note! I will fix it

Reply:
done